src/main/java/net/minecraftforge/common/crafting/CraftingHelper.java

```diff
@@ -218,7 +218,13 @@ public static ItemStack getItemStack(JsonObject json, JsonContext context)
             // Lets hope this works? Needs test
             try
             {
-                NBTTagCompound nbt = JsonToNBT.getTagFromJson(GSON.toJson(json.get("nbt")));
+                JsonElement element = json.get("nbt");
+                NBTTagCompound nbt;
+                if(element.isJsonObject())
```

Comment:
Yes for single line ifs, no {}s is fine. As long as you tab thing sin correctly. 
But for multi-line things, new lines for braces so that they are easily matched up. Its always a holy war on style, but this is the easiest i've found to read. However this is a clarification that is needed, If you go more then one level deep, Use braces. So while technically a if, right above a for, right above another if doesn't need braces, it should have them.